docs/wiki/frameworkx/sitting-framework.md

```diff
@@ -10,8 +10,11 @@ version:
   major: 3
   minor: 2
   patch: 0
+redirect_from: "/wiki/framework/sitting-framework.md"
 ---
 
+_Sitting has been moved from ACE3 (in v3.8.0) to ACEX._
+
 <div class="panel callout">
     <h5>Note:</h5>
     <p>ACE3 Sitting has been deprecated in favour of ACEX Sitting and will be removed in a future version.</p>
```

Comment:
This note should also be updated to be the same as in `featurex/sitting.md`.

Somehow I completely missed this framework page when I moved the feature page...

Reply:
Its the same on both. Should they not be the same?

Reply:
Actually, there is no note on the feature sitting page, I guess we removed that a few version later, can remove it from here as well then.